ycmd.el

```diff
@@ -567,6 +567,14 @@ _LEN is ununsed."
     (setq ycmd--on-focus-timer
           (run-at-time 1.0 nil #'ycmd--on-unparsed-buffer-focus))))
 
+(defmacro ycmd--with-all-ycmd-buffers (&rest body)
+  "Execute BODY with each `ycmd-mode' enabled buffer."
+  (declare (indent 0) (debug t))
```

Comment:
It shouldn't be removed. The `declare` form is a special form to add meta data to macros or functions.
See `(info "(elisp) Declare Form")`